src/models/apps/expense-split-app/expense.model.js

```diff
@@ -0,0 +1,67 @@
+import mongoose, { Schema } from "mongoose";
+import {
+  AvailableExpenseTypes,
+  AvailablePaymentMethods,
+  ExpenseTypes,
+  PaymentMethods,
+} from "../../../constants.js";
+
+const expenseSchema = mongoose.Schema(
```

Comment:
use `new Schema` instead of mongoose.Schema please go through the other models to get idea about the code style we are following